python/tvm/contrib/cutlass/gen_tensor_op.py

```diff
@@ -792,6 +801,12 @@ def get_batch_on_arg(arg_name, arg_shape):
         headers.append("cutlass/layout/matrix.h")
         attrs = {"input": func_args[0], "gamma": func_args[1], "beta": func_args[2]}
         attrs.update(dict(annotations))
+
+        if isinstance(attrs["M"], tvm.tir.Var):
```

Comment:
Does this mean we support batch?

Reply:
This is unrelated to this PR but needed for dolly. It enables layer norm offload with dynamic first dimension.